### PR TITLE
Skills

### PR DESCRIPTION
**Pull request by pawnow**

### CODE REVIEW

File: skills.pl [hunk 1]
```diff
@@ -0,0 +1,90 @@
+:- dynamic hero/1, lvl_bonus/4, stat_bonus/2, level/1.
+
+hero_class(dwarf).
+hero_class(archer).
+hero_class(knight).
+
+stat(strength).
+stat(defending).
```
Comment (prolativ) [inline skills.pl]: [grammar-nazi=on]I'd rather call it defense/defence[grammar-nazi=off]

File: skills.pl [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	retractall(level(_)),
+	assertz(level(LvlUp)),
+	write('Stats after leveling up: '), nl,
+	printCurrentStats.
+	
+try_choose_value_to_go_up(_) :-
+	write('Incorrect choice, try other option'), nl,
+	levelUp.
+
+printCurrentStats :-
```
Comment (prolativ) [inline skills.pl:76]: Add it also to commands and help menu.

File: game_init.pl [hunk 3]
```diff
@@ -27,7 +28,15 @@
 
 event(start_game, (
 	write("You wake up in the middle of the meadow with nothing in hands."), nl,
-	retractall(event(start_game, _))
+	retractall(event(start_game, _)),
+	assertz(event(choose_class, _))
+)).
+
+event(choose_class, (
```
Comment (prolativ) [inline game_init.pl]: Personally I would rather put it in restarting (starting new) game

File: skills.pl (hunk 2 above)
Comment (pawnow) [reply skills.pl:76]: I've added it to help and commands

File: skills.pl (hunk 1 above)
Comment (pawnow) [reply skills.pl]: You are right, I changed it to defence

File: game_init.pl (hunk 3 above)
Comment (pawnow) [reply game_init.pl]: Done

**Pull request merged**